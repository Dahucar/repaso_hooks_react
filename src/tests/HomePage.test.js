const { shallow } = require("enzyme");
const { HomePage } = require("../components/09-useContext/pages/HomePage");
const { UserContext } = require("../components/09-useContext/UserContext");

describe('Prueba en el <HomePage />', () => {
    const user = {
        name: 'Daniel',
        email: 'dan.gmail'
    }
    const wrapper = shallow( 
        <UserContext.Provider value={{ user }}>
            <HomePage />
        </UserContext.Provider>
    );
    test('Debe hacer match con la instantanea,', () => {
        expect( wrapper ).toMatchSnapshot();
    });
});
