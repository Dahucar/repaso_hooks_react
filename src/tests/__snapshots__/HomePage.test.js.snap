// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en el <HomePage /> Debe hacer match con la instantanea, 1`] = `<HomePage />`;
